packages/lime-app/files/96-lime-app-index_page

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+uci set uhttpd.main.index_page=lime_app_index.html
+uci commit uhttpd
```

Comment:
uci commit should not be necesary as the boot script runs uci commit after each uci-defaults file (https://github.com/openwrt/openwrt/blob/master/package/base-files/files/etc/init.d/boot#L17)

Reply:
thanks! I didn't know :)